src/calculateNodeHeight.js

```diff
@@ -34,10 +34,12 @@ const SIZING_STYLE = [
 
 let computedStyleCache = {};
 let hiddenTextarea;
+const hiddenTextareaId = 'react-textarea-autosize-textarea-for-calculations';
 
 export default function calculateNodeHeight(uiTextNode,
     useCache = false,
     minRows = null, maxRows = null) {
+  hiddenTextarea = document.getElementById(hiddenTextareaId);
```

Comment:
better check would be against `hiddenTextarea.parentNode` than polluting the global environment with custom id, would u like to revisit the code and make appropriate changes so we can merge it in?